GitCommands/Git/Commands/GitCommandHelpers.cs

```diff
@@ -17,26 +18,36 @@ public static ArgumentString CherryPickCmd(ObjectId commitId, bool commit, strin
             };
         }
 
-        public static ArgumentString SubmoduleUpdateCmd(string? name)
+        public static ArgumentString SubmoduleUpdateCmd(string? name, IEnumerable<GitConfigItem> configs = null)
```

Comment:
?
```suggestion
        public static ArgumentString SubmoduleUpdateCmd(string? name, IEnumerable<GitConfigItem>? configs = null)
```

Reply:
I am doing the innumerable on purpose. It is so 5fat if other configs are needed in future it's as simple as concat configs. The plan is to pass down these from allow from user in next. Adjusting ab innumerable to allow building needed configs is a lot better than passing strings around.

Reply:
In this case it was the nullability I commented on.
(I would actually preferred a string here, I see no reason to prepare for multiple settings now. I had to think a few seconds when seeing it and prefer the simple solution. But it was not enough to comment on it in the first place or to request changes.)